go/vt/log/log_test.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
nit: Copyright notice looks incorrectly formatted. This needs to be verbatim like the others to prevent poisoning (except for the year). Could you get this from a good source?